manifests/t/Tyrrrz/DiscordChatExporter/CLI/2.42.6/Tyrrrz.DiscordChatExporter.CLI.locale.en-US.yaml

```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Tyrrrz.DiscordChatExporter.CLI
+PackageVersion: 2.42.6
+PackageLocale: en-US
+Publisher: Tyrrrz
+PackageName: DiscordChatExporter.CLI
+License: GNU

```

Comment:
The license is MIT now

Reply:
Thanks, this PR has been around quite a while now 🥲